conf/application.conf

```diff
@@ -72,6 +72,11 @@ microservice {
       host = localhost
       port = 9250
     }
+
+    income-tax-subscription {
```

Comment:
Before this can be merged in we will need a config entry in app config base to enable this config in the environments